src/data/staking-products.json

```
@@ -381,6 +381,36 @@
         "eventName": "Clicked Allnodes go to link"
       }
     },
+    {
+      "name": "Ethpool",
+      "svgPath": "ethpool-glyph.svg",
+      "hue": 20,
+      "launchDate": "2020-11-02",
+      "url": "https://ethpool.org/",
+      "audits": [],
+      "minEth": 32,
+      "additionalStake": null,
+      "additionalStakeUnit": null,
+      "monthlyFee": null,
+      "monthlyFeeUnit": null,
+      "isFoss": false,
+      "hasBugBounty": false,
+      "isTrustless": false,
+      "isPermissionless": false,
+      "pctMajorityClient": 15,
```

I saw your comment that this was for the execution side... this currently refers to consensus clients, but [this issue](https://github.com/ethereum/ethereum-org-website/issues/9840) is discussing changing that. 

This also shines light on maybe needing to adjust this definition a little more, since it's a little gray now. It was intended to highlight services that were running a majority of their validators using the most popular CL client. Right now Lighthouse and Prysm are pretty much neck-and-neck (which is incredible compared to a year ago!)...

Given the current use and definition, I would suggest we change this to 0% for now, since you aren't using any Prysm clients (which are still slightly in the lead)... neither of these clients have a supermajority, but they each have the ability to halt finality. I might suggest (separately) that we update this to be "more than 50% of validators are using a client that holds > 33% network share" (both for execution clients and consensus clients)
```suggestion
      "pctMajorityClient": 0,
```